src/style.scss

```diff
@@ -101,7 +103,7 @@
     margin-left: 1rem;
   }
 
-  .details-slot {
+  .detailsSlot {
     padding: var(--auro-size-lg);
     padding-top: 0;
```

Comment:
Looks like there are two places .detailsSlot {} is declared and both define padding-top: 0.

Consider refining this to a single instance to reduce maintainability.

Reply:
The definitions are in separate contexts to override a base setting. Placing in a single selector will break the UI.